Dockerfile.Fedora.openshift

```diff
@@ -0,0 +1,28 @@
+FROM registry.fedoraproject.org/fedora:28
```

Comment:
Can we get away with just fedora rather than fedora:28?  Just to avoid bumping this per new version release.

Reply:
I specified :28 to avoid it automatically switching over to 29 when it becomes available.  They will be triggers that might build this at times we don't expect so this is safest imho